arrow.svg

```diff
@@ -0,0 +1,114 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<!-- Created with Inkscape (http://www.inkscape.org/) -->
+
+<svg
```

Comment:
This is very interesting! :) I was searching about `<svg>` when I was implementing the carousel. I was able to do a circle but not an arrow inside of it.

Reply:
Oops, I wasn't supposed to check this file in. I didn't like it and didn't use it. Too long! My code has a much shorter arrow svg. I'll delete this.

Reply:
So much code for an arrow. haha

Reply:
I drew the arrow in a graphics program and that's what it spit out...